### PR TITLE
Update README to reference smqtk-iqr

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,14 @@ Some above-mentioned packages supporting AI/ML topics include the following:
...
   that makes use of human feedback.
 
-These packages are related as followed:
+* [SMQTK-IQR] provides classes and utilities to perform the Interactive Query
+  Refinement (IQR) process. This package also includes a web API exposing the
+  use of these tools as well as an example web UI service to demonstrate the
+  capability. These services are additionally containerized to provide some
+  portability of these services.
+
+These packages are related as followes:
```
Comment (joshanderson-kw) [inline README.md:59]:
```suggestion
These packages are related as follows:
```

Comment (Purg) [reply README.md:59]: Oops, fix incoming.

File: docs/release_notes/pending_release.rst [hunk 2]
```diff
@@ -39,6 +39,8 @@ Miscellaneous
 * Add script to help with updating versioning and updating changelog during
   the release process
 
+* Updated README to include reference to the SMQTK-IQR package.
```
Comment (joshanderson-kw) [inline docs/release_notes/pending_release.rst:42]: More of a question than an issue, but I noticed the pending release notes files use both past and present tense (`Add` and `Updated`) versus all present (`Add` and `Update`) or all past tense (`Added` and `Updated`). Do you have a preference on using past vs. present tense in these files?

Comment (Purg) [reply docs/release_notes/pending_release.rst:42]: Yeah, I've never been the best at this, nor at enforcing this with others. On this particular point I would say I'm preferential to the past tense, but I'm not overly married to it as I realize there are perspective justifications for both angles. I can add a card to our board to normalize the tenses used across change notes, or at least in the current pending files, and to add verbiage to the contributing file.

**Review by joshanderson-kw — APPROVED** [review]

**Pull request merged**